### ISSUE TITLE
Arm?

### ISSUE THREAD

**Issue by shandshellin** [PhasecoreX/docker-ut2004-server#21]
Does this image work on any arm processor?

**Comment by PhasecoreX** [comment]
No, it only works with x86. Unlike ut99, which has the community working on it and making an arm version, ut2004 here is just the latest version that Epic released almost 2 decades ago.

I could look into something like box86, not sure how fast that would run. Also I think that only runs on 32bit arm devices, so maybe not worth the hassle. If ut2004 had a 64bit x86_64 version, box64 would be much more interesting.

**Comment by PhasecoreX** [comment]
Just a 2.5 year update on this, OldUnreal is making a patch for UT2004 (like they did with UT99) which should support arm64. Once they release it, I will be updating this to support arm64 as well!
